javatests/arcs/sdk/wasm/UnicodeTest.kt

```diff
@@ -19,7 +19,8 @@ class UnicodeTest : AbstractUnicodeTest() {
         val pass = if (handle.name == "sng") {
             ((handle as WasmSingletonImpl<*>).fetch() as UnicodeTest_Sng).pass
         } else {
-            ((handle as WasmCollectionImpl<*>).iterator().next() as UnicodeTest_Col).pass
+            // TODO(cypher1): This is unsafe, there is no guarantee of ordering.
+            ((handle as WasmCollectionImpl<*>).fetchAll().iterator().next() as UnicodeTest_Col).pass
```

Comment:
I'm not too familiar with this test case, but I suspect we only expect there to be a single element in the collection.

If that's true, I think you can use `.single()` to return the single element or throw: `handle.fetchAll().single()`